resident/resident-service/src/main/java/io/mosip/resident/constant/ApiName.java

```diff
@@ -36,6 +36,8 @@
 
 	IDAUTHREVOKEVID,
 
+	IDAUTHDEACTIVATEVID,
```

Comment:
If this API is not used anywhere in resident service, lets not add it
Please remove in application-local.properties also if you are removing here